solidity/contracts/TestFakeGravity.sol

```diff
@@ -0,0 +1,15 @@
+//SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.10;
+
+import "hardhat/console.sol";
+import "./Gravity.sol";
+
+contract TestFakeGravity is Gravity {
```

Comment:
a commetn here describing why a fake gravity is needed would be good. I understand what's going on, but a reader in the future probably wont.

Reply:
Comment added